__tests__/asset-proxy.ts

```diff
@@ -0,0 +1,75 @@
+import { http } from 'msw'
+
+import {
+  currentTestEnvironment,
+  expectIsPlaceholderResponse,
+} from './__utils__'
+
+beforeEach(() => {
+  globalThis.server.use(
+    http.get('https://whatever.org/image', () => {
+      return new Response('', {
+        headers: {
+          'content-type': 'image/png',
+          'Set-Cookie':
+            'sessionId=abc123; Expires=Wed, 09 Nov 2024 07:28:00 GMT; Path=/; Domain=whatever.org; Secure; HttpOnly; SameSite=None',
+        },
+      })
+    }),
+    http.get('https://whatever.org/notimage', () => {
+      return new Response('', {
+        headers: { 'content-type': 'application/json' },
+      })
+    }),
+  )
+})
+
+test('request to https://asset-proxy.serlo.org/src?url=* gets asset from url query parameter', async () => {
```

Comment:
@hugotiburtino what do you think, should be use `https://asset-proxy.serlo.org/src?url=*` or just `https://asset-proxy.serlo.org/?url=*`?

Reply:
Or `https://asset-proxy.serlo.org/img?url=*` might make sense, then we could maybe add other types later.

Reply:
I think `https://asset-proxy.serlo.org/img?url=*` would be the most descriptive